pkg/conf/conf.go

```diff
@@ -88,6 +88,21 @@ func SafeConfig(n *node.Node, fp string) error {
 	return tree.Export(n, "yaml", fp)
 }
 
+func GetExporters2(configFp string) (map[string]Exporter, error) {
+	err := LoadHarvestConfig(configFp)
+	if err != nil {
+		return nil, err
+	}
+	exporters := Config.Exporters
+
+	if exporters == nil {
```

Comment:
why is it an error if exporter section not found? It should just be nil right?

Reply:
actually as part of this PR, just changed node to struct, logic wise it's same as earlier as GetExporters.